runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -386,6 +460,12 @@ bool J9::RecognizedCallTransformer::isInlineable(TR::TreeTop* treetop)
       case TR::java_lang_StrictMath_sqrt:
       case TR::java_lang_Math_sqrt:
          return comp()->target().cpu.getSupportsHardwareSQRT();
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+      case TR::java_lang_invoke_Invokers_checkExactType:
+         return true;
+      case TR::java_lang_invoke_Invokers_checkCustomized:
+         return node->getChild(0)->getSymbolReference()->hasKnownObjectIndex();
```

Comment:
During inlining, the node may not have known object index. The known object info can come from inlined stack. Check if we're during inlining, and check the prex arg from the inline stack
```
TR_PrexArgInfo *argInfo = comp->getCurrentInlinedCallArgInfo();
   TR_PrexArgument *arg = argInfo->get(0);
   if (arg && arg->getKnownObjectIndex() != TR::KnownObjectTable::UNKNOWN)
      {   
```